s:789:"<b>Test 1: Teste 1 </b><br />Incorrect program result
<br /> --- Input ---
<br /><pre><i> 5
10
20
30
40
50
1
2
3
4
5
</i></pre>
<br /> --- Program output ---
<br /><pre><i>Numero de elementos:Valores da lista:Valores da lista:Valores da lista:Valores da lista:Valores da lista:Valores da lista:Valores da lista:Valores da lista:Valores da lista:Valores da lista:30.000000
Traceback (most recent call last):
  File "estatistica.py", line 35, in &lt;module&gt;
    print ('%2f'%dp_a)
NameError: name 'dp_a' is not defined
</i></pre>
<br /> --- Expected output (text)---
<br /><pre><i> '30.00'
'15.81'
'3.00'
'1.58'
</i></pre>
<br /><b>Summary of tests</b><br /><pre><i>+------------------------------+
|  1  test run/ 0 tests passed |
+------------------------------+
</i></pre>
<br /><br />";